### DISCUSSION TITLE
Not received Student Developer pack

### DISCUSSION THREAD

**Discussion by rohitdeka-1** [community/community#147137] [GitHub Education] [answered]
### Hi everyone,

I got approved for my student developer pack on 12th and it says I will receive it within 3 days its almost 16th of dec , actually I am quite excited to get it XD . Someone Help please : )
![Screenshot 2024-12-15 160935](https://github.com/user-attachments/assets/06c7dbc0-a76f-48be-bb13-cadaa79f19ca)

**Comment by saidibyanshuiitbcse** [comment]
same with me also
i also approved on 12 dec

**Comment by davevad93** [comment, marked as answer]
Hi @rohitdeka-1 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @rohitdeka-1 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
